Makefile

```diff
@@ -627,6 +658,8 @@ $(SYSROOT_LIB)/libsetjmp.a: $(LIBSETJMP_OBJS)
 
 $(PIC_OBJS): CFLAGS += -fPIC -fvisibility=default
 
+$(LIBC_NONLTO_OBJS): CFLAGS += -fno-lto
```

Comment:
nit: I wonder if we could instead filter out the `-flto` flag instead of passing both `-flto` and `-fno-lto`?

Reply:
done